### PR TITLE
fix wallpaper category

### PR DESCRIPTION
**Pull request by kziemianek**
##### Description
*it should be listed in for_fun category

### CODE REVIEW

File: app/dashboard/views.py [hunk 1]
```diff
@@ -871,6 +871,7 @@ def toolbox(request):
     profile_up_votes_tool_ids = ','.join(str(x) for x in ups)
     downs = list(ToolVote.objects.filter(profile_id = profile_id, value = -1).values_list('tool', flat=True))
     profile_down_votes_tool_ids = ','.join(str(x) for x in downs)
+    print(profile_up_votes_tool_ids)
```
Comment (owocki) [inline app/dashboard/views.py]: did you mean to keep this in there?

**Comment by owocki** [comment]
migration LGTM

File: app/dashboard/views.py (hunk 1 above)
Comment (kziemianek) [reply app/dashboard/views.py]: no! :)

**Comment by mbeacom** [comment]
@kziemianek Thanks for this! - I was in the middle of updating some of the `Tool` related models/views when this came in. I opted to include the migration in the other PR (#944) to avoid an additional `migration` file.  If you're strongly opposed to that, I can merge this first... But I figured it would make sense to consolidate them.

**Pull request closed without merging**